src/Compilers/CSharp/Portable/Symbols/Source/CustomModifierUtils.cs

```diff
@@ -35,8 +35,7 @@ internal static class CustomModifierUtils
             // have already been compared.
             MethodSymbol constructedSourceMethod = sourceMethod.ConstructIfGeneric(destinationMethod.TypeArguments);
 
-            customModifiers = CustomModifiersTuple.Create(constructedSourceMethod.ReturnTypeCustomModifiers,
-                                                          destinationMethod.ReturnsByRef ? constructedSourceMethod.RefCustomModifiers : ImmutableArray<CustomModifier>.Empty);
+            customModifiers = CustomModifiersTuple.Create(constructedSourceMethod.ReturnTypeCustomModifiers, constructedSourceMethod.RefCustomModifiers);
```

Comment:
ReturnsByRef is an api that returns true for methods that return by reference. There is another API now (ReturnsByRefReadOnly) that needs copying as well. My understanding is that **all** such implementations should exactly duplicate their overridden methods (CLR-spec), which the new implementation is doing.

---
In reply to: [118068064](https://github.com/dotnet/roslyn/pull/19658#discussion_r118068064) [](ancestors = 118068064)